video.js

```diff
@@ -381,6 +375,17 @@ define(['./teletext', './utils'], function (Teletext, utils) {
                 // Handle HSync
                 if (this.inHSync) this.handleHSync();
 
+                // Handle latching of vertical adjust pending.
```

Comment:
Thanks for the clear comment! This wasn't as scary a change as I was anticipating!